scripts/pinned_build.sh

```diff
@@ -0,0 +1,136 @@
+#! /usr/bin/bash
+
+echo "Mandel Pinned Build"
+
+if [[ $NAME != "Ubuntu" ]]
+   then
+      echo "Currently only supporting Ubuntu based builds. Proceed at your own risk."
+fi
+
+if [ $# -eq 0 ] || [ -z "$1" ]
+   then
+      echo "Please supply a directory for the build dependencies to be placed and a directory for mandel build and a value for the number of jobs to use for building."
+      echo "The binary packages will be created and placed into the mandel build directory."
+      echo "./pinned_build.sh <dependencies directory> <mandel build directory> <1-100>"
+      exit -1
+fi
+
+CORE_SYM=EOS
+DEP_DIR=$1
+MANDEL_DIR=$2
+JOBS=$3
+CLANG_VER=11.0.1
+BOOST_VER=1.70.0
+LLVM_VER=7.1.0
```

Comment:
How did these versions come about? Some of them are older than what was used in the 2.0.x pinned builds even.

Reply:
Feel free to recommend versions. I honestly didn't really look at the B1 stuff as personally I would've liked the pinned packages to use as low of a version as we support without running into issues or performance degradation as they should be the most stable at this point.

I know the clang version is a bit high, but the delta was noticeable enough that I wanted that one to be at 11.

Reply:
It's a bit unexpected because the versions of LLVM & boost chosen are over three years old, older than we used in 2.0.x, and older than what ships in Ubuntu 20.04 (our only supported version for 3.1). But yes they are compatible so there isn't anything really 'wrong' with it, just curious what the criteria was for picking them.

FWIW there are parts of the code that are known to perform notably better with newer version of boost, for example EOSIO/eos#8147 (which uhhh why is that not in mandel)